
<html>

<head>
<meta http-equiv="Content-Language" content="zh-cn">
<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
<title>TargetLBS</title>

<style>
<!--
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	font-size:10.5pt;
	font-family:"Times New Roman";
	margin-left:0cm; margin-right:0cm; margin-top:0cm}
-->
</style>
</head>

<body>

<div align="center">

<center>
<img src="http://zhanglab.ccmb.med.umich.edu/LS-align/logo/LSalign_Logo.png" border="0" width=600>
</center>

<table border="0" width="800" id="table1" bgcolor="#FFFFFF">
	<tr>
		<td valign="baseline" colspan="3">
		<p align="justify" style="line-height: 150%; margin-top: 10px">
		<font face="Georgia" size="2">
			  LS-align is an algorithm designed for atom-level structural comparison of ligand molecules.
				The target function of LS-align is a combination of inter-atom distance, atom mass, and chemical
				bond connections; while the final atom-to-atom alignment is generated by maximizing such 
				target function through an enhanced-greedy based, iterative heuristic search algorithm.
				LS-align program contains two modules: <i>Rigid-LS-align</i> for rigid-body ligand structure 
				comparison; and <i>Flexi-LS-align</i> for flixible structure comparison. 
				In particular, the <i>Flexi-LS-align</i> module seeks for optimal alignments of various alternative 
				conformers of the ligand molecules by rotating flexible bond-angles, 
				which allows the consideration of binding-induced conformational changes
				in the ligand structural comparison and alignment. 
		</font>
		</p>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3" bgcolor="#C0C0C0">
			<p><b>
			<font face="Georgia" size="2">
				<a name="Published Papers">Standalone TargetLBS program</a>
			</font>
			</b></p>
		</td>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3"><br/><font face="Georgia" size="2"><b><i>Journal Papers</i></b></font></td>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3">
		<ul>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Zi Liu, Dong-Jun Yu, and Yang Zhang. 
			    <u>LS-align: an atom-level, flexible ligand structural alignment algorithm for high-throughput virtual screening</u> [J]. <b>Bioinformatics</b>, 2018 (In press).
					[<a href="papers/LS-align.pdf" target=_blank>PDF</a>]
					[<a href="https://doi.org/10.1093/bioinformatics/bty081" target=_blank>DOI</a>]
					[<a href="https://zhanglab.ccmb.med.umich.edu/LS-align/" target=_blank>Web Server</a>]</font></li>
		  <li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Yang Li, Yang Zhang, and Dong-Jun Yu. 
			    <u>ATPbind: accurate protein-ATP binding site prediction by combining sequence-profiling and structure-based comparisons</u> [J]. <b>Journal of Chemical Information and Modeling</b>, 2018, 58: 501-510.
					[<a href="papers/ATPbind.pdf" target=_blank>PDF</a>]
					[<a href="https://zhanglab.ccmb.med.umich.edu/ATPbind/" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Yang Li, Ming Zhang, Xibei Yang, 
			Hong-Bin Shen, and Dong-Jun Yu. <u>Predicting Protein-DNA Binding Residues by Weightedly Combining Sequence-based Features and Boosting Multiple SVMs</u> [J]. <b>IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2017, 14(6): 1389-1398.
					[<a href="papers/TargetDNA.pdf" target=_blank>PDF</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetDNA" target=_blank>Web Server</a>]
					[<a href="dataset/TargetDNA_dataset.rar" target=_blank>Dataset</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Ke Han, Yang Li, Jing-Yu Yang, 
			Hong-Bin Shen, and Dong-Jun Yu. <u> TargetCrys: protein crystallization prediction by fusing multi-view features with two-layered SVM</u> [J]. <b>Amino Acids</b>, 2016, 48(11): 2533-2547. 
					[<a href="papers/2016-Amino-Acids.pdf" target=_blank>PDF 1</a>][<a href="papers/TargetCrys.pdf" target=_blank>PDF 2</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetCrys" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Yang Li, Wu-Xia Yan, Jing-Yu Yang, 
			Hong-Bin Shen, and Dong-Jun Yu. <u> KNN-based Dynamic Query-Driven Sample Rescaling 
				Strategy for Class Imbalance Learning</u> [J]. <b>Neurocomputing</b>, 2016, 191: 363-373. 
					[<a href="papers/TargetNUCs.pdf" target=_blank>PDF</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetNUCs" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Yang Li, Jing-Yu Yang, 
			Hong-Bin Shen, and Dong-Jun Yu. <u>GPCR-drug Interactions Prediction 
				Using Random Forest with Drug-Association-Matrix-Based Post-Processing 
				Procedure</u> [J]. <b>Computational Biology and Chemistry</b>, 2016, 60: 59-71. 
					[<a href="papers/2016-CBAC.pdf" target=_blank>PDF</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetGDrug" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Xue He, Ke Han, <b>Jun Hu</b>, Hui Yan, 
			Jing-Yu Yang, Hong-Bin Shen, and Dong-Jun Yu. <u>TargetFreeze: 
			Identifying Antifreeze Proteins via a Combination of Weights using 
			Sequence Evolutionary Information and Pseudo Amino Acid Composition</u>.
			<b>Journal of Membrane Biology</b>, 2015, 248(6): 1005-1014.
					[<a href="papers/2015-JMBI.pdf" target=_blank>PDF</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetFreeze" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, <b>Jun Hu</b>, Qian-Mu 
			Li, Zhen-Min Tang, Jing-Yu Yang, and Hong-Bin Shen. <u>Constructing 
			Query-Driven Dynamic Machine Learning Model with Application to 
			Protein-Ligand Binding Sites Prediction</u>. <b>IEEE Transactions on 
			NanoBioscience</b>, 2015, 14(1): 45-58. 
					[<a href="papers/OSML.pdf" target=_blank>PDF</a>]
					[<a href="http://www.csbio.sjtu.edu.cn/bioinf/OSML" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, Yang Li, <b>Jun Hu</b>, Hong-Bin Shen, and Jing-Yu Yang.
			<u>Disulfide Connectivity Prediction with Modelled Protein 3D Structural Information and Random Forest Regression</u>. 
			<strong>IEEE/ACM Transactions on Computational Biology and Bioinformatics</strong>, 2015, 12(3): 611-621.
					[<a href="papers/2015-TCBB.pdf" target=_blank>PDF</a>]
					[<a href="http://csbio.njust.edu.cn/bioinf/TargetDisulfide" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2"><b>Jun Hu</b>, Xue He, Dong-Jun Yu*, 
			Xi-Bei Yang, Jing-Yu Yang, and Hong-Bin Shen. <u>A New Supervised 
			Over-Sampling Algorithm with Application to Protein-Nucleotide 
			Binding Residues Prediction</u>. <b>PLOS ONE</b>, 2014, 9(9): e107676
					[<a href="papers/2014-PLOS0NE.pdf" target=_blank>PDF</a>]
					[<a href="http://www.csbio.sjtu.edu.cn/bioinf/TargetSOS" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, <b>Jun Hu</b>, Hui Yan, Hong-Bin Shen,
             and Jing-Yu Yang. <u>Enhancing Protein-Vitamin Binding Residues Prediction 
             by Multiple Heterogeneous Subspace SVMs Ensemble</u>. <strong>BMC Bioinformatics</strong>, 2014, 15:297.
             [<a href="papers/TargetVita.pdf" target=_blank>PDF</a>]
             [<a href="http://www.csbio.sjtu.edu.cn/bioinf/TargetVita" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 20px; margin-top: 10px" align="justify">
			<font face="Georgia"><span style="font-size: 10pt">Dong-Jun Yu, <b>Jun Hu</b>, Jing 
			Yang, Hong-Bin Shen, Jinhui Tang, and Jing-Yu Yang. <u>Designing template-free predictor for targeting protein-ligand 
			binding sites with classifier ensemble and spatial clustering</u>.
			<b>IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>,&nbsp;</span></font><font size="2" face="Georgia">2013, 
			10(4): 994-1008.
						[<a href="papers/TargetS.pdf" target=_blank>PDF 1</a>]
						[<a href="papers/TargetS-new.pdf" target=_blank>PDF 2</a>]
						[<a href="http://www.csbio.sjtu.edu.cn/bioinf/TargetS" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 150%; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, <b>Jun Hu</b>, Xiaowei Wu, 
			Hong-Bin Shen, Jun Chen, Zhenmin Tang, Jian Yang, and Jingyu Yang.
			<u>Learning Protein Multi-View Features in Complex Space</u>, &nbsp;<b>Amino 
			Acids</b>.
			2013, 44(5): 1365-1379.[<a href="papers/COMSPA.pdf" target=_blank>PDF</a>][<a href="http://www.csbio.sjtu.edu.cn/bioinf/COMSPA" target=_blank>
              Web Server</a>]</font></li>
			<li>
			<p style="line-height: 150%; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, <b>Jun Hu</b>, Yan Huang, 
			Hong-Bin Shen, Yong Qi, Zhen-Min Tang and Jing-Yu Yang. <u>TargetATPsite: A Template-free Method for ATP Binding Sites 
			Prediction with Residue Evolution Image Sparse Representation and 
			Classifier Ensemble</u>.&nbsp;<b>Journal of Computational Chemistry</b>, 2013, 34: 974-985.&nbsp; 
			(<b> Inside Cover Story</b>)
						[<a href="papers/TargetATPsite.pdf" target=_blank>PDF</a>]
						[<a href="http://www.csbio.sjtu.edu.cn/bioinf/TargetATPsite" target=_blank>Web Server</a>]</font></li>
			<li>
			<p style="line-height: 150%; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu, <b>Jun Hu</b>, Zhen-Min Tang, 
			Hong-Bin Shen, Jian Yang, and Jing-Yu Yang. <u>Improving 
	Protein-ATP Binding Residues Prediction by Boosting SVMs with Random 
	Under-Sampling</u>. <b>Neurocomputing</b>, 2013, 104: 180-190.
						[<a href="papers/nerocomputing-2013.pdf" target=_blank>PDF</a>]
						[<a href="http://www.csbio.sjtu.edu.cn/bioinf/targetATP" target=_blank>Web Server</a>]</font></li>
		</ul>
		</td>
	</tr>	
	<tr>
		<td width="1178" valign="baseline" colspan="3"><br/><font face="Georgia" size="2"><b><i>Conference Papers</i></b></font></td>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3">
		<ul>
			
			<li>
			<p style="line-height: 150%; margin-top: 10px" align="justify">
		<font face="Georgia" size="2"><b>Jun Hu</b>, Zi Liu, and Dong-Jun Yu*. <u>Enhancing Protein-ATP and Protein-ADP Binding Sites Prediction Using Supervised Instance-Transfer Learning</u>. <b>The 4th Asian Conference on Pattern Recognition (ACPR 2017)</b>. 2017: 759-763 
						[<a href="papers/MS-ATPADPsite.pdf" target=_blank>PDF</a>]</font></li>
			
			<li>
			<p style="line-height: 150%; margin-top: 10px" align="justify">
			<font face="Georgia" size="2">Dong-Jun Yu,<b> Jun Hu,</b> Jian-Hua Xie, Yong Qi, and 
	Zhen-Min Tang. <u>Supervised Kernel Self-Organizing Map</u>. <b>2012 
	Sino-foreign-interchange Workshop on Intelligence Science and Intelligent 
	Data Engineering (IScIDE 2012)</b>. Springer, Nanjing, LNCS 7751, pp. 
	246--253. Springer, Heidelberg (2013). 
						[<a href="papers/IScIDE-2012.pdf" target=_blank>PDF</a>]</font></li>
		</ul>
		<p><font face="Georgia" size="2">[<a href="#Jun_Hu">Back</a>]</font>
		</td>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3" bgcolor="#C0C0C0">
			<p><b><font face="Georgia" size="2"><a name="Quick Links">Quick Links</a></font></b></p>
		</td>
	</tr>
	<tr>
		<td width="1178" valign="baseline" colspan="3">
			<ul>
				<li><a href="http://csbio.njust.edu.cn" target=_blank>D.J. YuLab</a></li>
				<li><a href="https://zhanglab.ccmb.med.umich.edu/" target=_blank>Yang ZhangLab</a></li>
				<li><a href="https://leeyang.github.io" target=_blank>Yang Li</a></li>
				<li><a href="https://heyan-svm.github.io/" target=_blank>He Yan</a></li>
			</ul>
			 
			<p><font face="Georgia" size="2">[<a href="#Jun_Hu">Back</a>]</font>
		</td>
	</tr>
	</table>
</div>

</body>
</html> 
